src/patterns/components/stepper/react/info/base.hbs

```diff
@@ -0,0 +1,95 @@
+<p class="sprk-u-mbm">
+  See below for React properties for the <code>SprkStepper</code> component.
+</p>
+
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+      <td>string</td>
+      <td>A space-delimited list of additional CSS classes that should be applied to the Stepper.</td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">hasDarkBackground</td>
+      <td>bool</td>
+      <td>If true, the Stepper will be rendered on a dark background. <code>($sprk-blue)</code></td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">idString</td>
+      <td>string</td>
+      <td>The value supplied will be assigned to the <code>data-id</code> attribute on the Stepper.</td>
+    </tr>
+  </tbody>
+</table>
+
+<p class="sprk-u-mbm">
+  See below for React properties for the <code>SprkStepperStep</code> component.
+</p>
+
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">additionalClasses</td>
+      <td>string</td>
+      <td>A space-delimited list of additional CSS classes that should be applied to the StepperStep.</td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">idString</td>
+      <td>string</td>
+      <td>The value supplied will be assigned to the <code>data-id</code> attribute on the StepperStep.</td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">analyticsString</td>
+      <td>string</td>
+      <td>The value supplied will be assigned to the <code>data-analytics</code> attribute on the StepperStep.</td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">title</td>
+      <td>string</td>
+      <td>The title of the step</td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">isSelected</td>
+      <td>bool</td>
+      <td>If true, this step will be selected.</td>
```

Comment:
This needs a little bit more info. I would add that it will be selected when the component mounts the first time. But, once another step is clicked, the 'isSelected' no longer applies.